Infrastructure.DataAccess/Mapping/OrganizationMap.cs

```diff
@@ -10,7 +10,7 @@ public OrganizationMap()
         {
             // Properties
             Property(x => x.Name)
-                .HasMaxLength(100)
+                .HasMaxLength(Organization.MaxNameLength)
```

Comment:
den der rettelse kan vi vel godt beholde for den var der i forvejen bare hard coded